lib/modules/@apostrophecms/schemas/src/apos/components/AposInputCheckboxes.stories.js

```diff
@@ -1,6 +1,6 @@
 import { storiesOf } from '@storybook/vue';
 
-import AposInputCheckbox from './AposInputCheckbox.vue';
+import AposInputCheckboxes from './AposInputCheckboxes.vue';
```

Comment:
You should go ahead and remove imports of Apostrophe Vue components because they all get registered in Vue by name automatically.